### PR TITLE
Disable out of stock items

### PR DESCRIPTION
**Pull request by Jpablogs8**
- Added About button
- Added About info views for portrait and Landscape
- Modified rewards views
- Fixed bug for campaigns in

### CODE REVIEW

File: Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs [hunk 1]
```diff
@@ -100,17 +101,21 @@ public void LoadImage()
...
-                                url = TallURL;
-                            }
-                            else if (!IsTall && !string.IsNullOrEmpty(SmallURL))
-                            {
-                                url = SmallURL;
-                            }
-                            else
-                            {
-                                DisplayAd = false;
+                                if (IsTall && !string.IsNullOrEmpty(TallURL))
```
Comment (markgrossnickle) [inline Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs:106]: Need to check if it is portrait mode here. Should only show TallURL in landscape. This was an existing bug.  @Jpablogs8

File: Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs [hunk 2]
```diff
@@ -494,17 +499,27 @@ public void DisplayCurrentImage()
...
                     }
-                    if(Data.Texture && (displayImage.sprite == null || Data.Texture != displayImage.sprite.texture)) AdImage.sprite = Data.Texture.ToSprite();
+                    if(Data.Texture && (displayImage.sprite == null || Data.Texture != displayImage.sprite.texture))
+                    {
+                        AdImage.sprite = Data.Texture.ToSprite();
+                    }
+                    else 
+                    {
+                        AdImage.sprite = displayImage.sprite.texture.ToSprite();
+                    }
```
Comment (markgrossnickle) [inline Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs:517]: Does this ever happen? Where we are trying to load an Ad but it doesn't show?

**Review by markgrossnickle — CHANGES_REQUESTED** [review]
Looks good! Just a couple questions and one bug fix

File: Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs (hunk 2 above)
Comment (Jpablogs8) [reply Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs:517]: I forgot to remove it, it was to test something.

File: Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs (hunk 1 above)
Comment (Jpablogs8) [reply Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs:106]: Is better this?

if (IsTall && !string.IsNullOrEmpty(TallURL) )
{
        if(!ScutiUtils.IsPortrait())
               url = TallURL;

}

Comment (markgrossnickle) [reply Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs]: Actually.... I just realized we don't need this.  We already check IsTall which is probably never true in portrait. 

lol

Sorry!  I'll strip this out and merge

Comment (Jpablogs8) [reply Scuti/Scripts/UI/Store/OfferSummaryPresenterBase.cs]: Yes, indeed istall always arrives False.

**Pull request merged**